### PR TITLE
closes #21

### LINKED ISSUE #21: Bug in Yoneda Embedding functor (when applied on zero morphisms)
The following code:
```
LoadPackage( "FunctorCategories" );
Qq := PathAlgebra( HomalgFieldOfRationals(), RightQuiver( "q(2)[x:1->2]" ) );
Qq_oid := Algebroid( Qq );
Y := YonedaEmbedding( Qq_oid );
Y( ZeroMorphism( Qq_oid.1, Qq_oid.2 ) );
```
raises the following error:
```
Error, source and target of the zero path is ambiguous
 at /home/kamal/.gap/pkg/Algebroids/gap/Algebroids.gi:1264
```

### PR DESCRIPTION
**Pull request by kamalsaleh**
The issue can be fixed by using the main algebroids-morphism constructor.

### CODE REVIEW

**Comment by kamalsaleh** [comment]
Hey @zickgraf, do you have any idea what might be wrong?

**Comment by zickgraf** [comment]
Yes, just apply PackageJanitor, then everything should be fine again :-)

**Comment by mohamed-barakat** [comment]
I applied PackageJanitor and the tests are still failing :(

**Comment by zickgraf** [comment]
The PR has to be rebased to pick up the changes made by applying PackageJanitor. And it seems like you have applied an old version of PackageJanitor. This also fixes the error but not in the intended way :D

**Comment by mohamed-barakat** [comment]
Thanks for the hint @zickgraf and for the PR @kamalsaleh.

File: PackageInfo.g [hunk 1]
```diff
@@ -15,7 +15,7 @@ Version := Maximum( [
                    ## this line prevents merge conflicts
                    "2018.10-18", ## Florian's version
                    ## this line prevents merge conflicts
-                   "2020.05-10", ## Kamal's version
+                   "2021.02-15", ## Kamal's version
```
Comment (mohamed-barakat) [inline PackageInfo.g:18]: Just for the future. This numbering scheme is a bit different. The last number should reflect the number of the commit in the current month, not a date.

**Pull request merged**